lncfg/wtclient.go

```diff
@@ -6,7 +6,8 @@ import "fmt"
 type WtClient struct {
 	// Active determines whether a watchtower client should be created to
 	// back up channel states with registered watchtowers.
-	Active bool `long:"active" description:"Whether the daemon should use private watchtowers to back up revoked channel states."`
+	// Note: Default value is true.
+	Active bool `long:"active" description:"(Deprecated) Whether the daemon should use private watchtowers to back up revoked channel states. This will always be set to true by default, and is here for backwards compatibility."`
```

Comment:
Should we change the text on this to not confuse users? This could be misunderstood as "I don't need to do anything anymore, my node will automatically back up with Watchtowers", while only the basic feature to interact with towers is turned on by default. So perhaps something along the lines of "turn on watchtower client support and allow private towers to be configured and added manually"?

The same goes for the new `Deactivate` flag, just with opposite phrasing.